pkg/rc/fields/fields.go

```diff
@@ -32,8 +33,10 @@ type RC struct {
 	// A set of labels that will be added to every pod scheduled by this controller
 	PodLabels labels.Set
 
-	// The desired number of instances of the manifest that should be scheduled
-	ReplicasDesired int
+	// The desired number of instances of the manifest that should be
+	// scheduled. A pointer is used here to safely distinguish between the
+	// count being 0 and not being present in the RC json
+	ReplicasDesired *int
```

Comment:
well, if we never expect a `nil` in a Raw RC, and we panic if we try to unmarshal such a one, doesn't it mean that in the cooked/parsed RC we should expect that this will never be `nil`, and thus can keep it a plain `int`?